packages/alfa-rules/test/sia-r69/rule.spec.tsx

```diff
@@ -418,7 +418,8 @@ test("evaluate() is inapplicable to text that would otherwise fail if it was not
   const document = h.document([
     <html
       style={{
-        backgroundImage: "url('foo.png')",
+        backgroundImage: "linear-gradient(#000 50%, transparent 50%)",
+        backgroundColor: "#000",
         color: "#000",
```

Comment:
Wouldn't that be inapplicable as invisible because it's black-on-black?

Reply:
For the record, this is answered in above answer